evm-node.md

```diff
@@ -215,9 +215,9 @@ Because the same account may be loaded more than once, implementations of this i
 
 ```{.k .node}
     syntax KItem ::= vmResult ( return: String , gas: Int , refund: Int , status: Int , selfdestruct: List , logs: List , AccountsCell , touched: List )
-    syntax KItem ::= extractConfig ( GeneratedTopCell ) [function, symbol]
- // ----------------------------------------------------------------------
-    rule extractConfig ( <generatedTop>
+    syntax KItem ::= extractConfig ( KevmCell ) [function, symbol]
+ // --------------------------------------------------------------
+    rule extractConfig ( <kevm>
```

Comment:
I'm fairly suspicious of this change, unless we intend to use the ocaml node in the future instead of the llvm node, which seems unlikely.

Reply:
@dwightguth when I switch back to GeneratedTop for this whole thing, it fails to build. Probably because the Node is built with the OCaml backend?